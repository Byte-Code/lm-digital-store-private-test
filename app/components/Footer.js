import React from 'react';
import styled from 'styled-components';
import { Link } from 'react-router';

const Wrapper = styled.div`
  position: fixed;
  background: transparent;
  padding: 0 10px 10px;
  bottom: 0;
`;

const Button = styled.div`
  width: 260px;
  height: 62px;
  background: #333333;
  color: #fff;
  cursor: pointer;
`;

const Footer = () => (
  <Wrapper>
    <Link to="/world">
      <Button>Homepage Mondo</Button>
    </Link>
  </Wrapper>
);

export default Footer;
