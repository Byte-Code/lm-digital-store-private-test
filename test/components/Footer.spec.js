import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../../app/components/Footer';

describe('Footer', () => {
  it('should render properly', () => {
    const result = shallow(<Footer />);
    expect(result).toMatchSnapshot();
  });
});
