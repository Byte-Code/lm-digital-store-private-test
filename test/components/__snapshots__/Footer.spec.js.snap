// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render properly 1`] = `
<styled.div>
  <LinkBack>
    <styled.div>
      <NavigationArrowBack
        color="#fff"
        style={
          Object {
            "height": 50,
            "width": 50,
          }
        }
      />
      <p>
        Indietro
      </p>
    </styled.div>
  </LinkBack>
  <Link
    onlyActiveOnIndex={false}
    style={Object {}}
    to="/world"
  >
    <styled.div>
      <ActionHome
        color="#fff"
        style={
          Object {
            "height": 50,
            "width": 50,
          }
        }
      />
      <p>
        Homepage
      </p>
    </styled.div>
  </Link>
  <TutorialButton>
    <styled.div>
      <ActionHelpOutline
        color="#fff"
        style={
          Object {
            "height": 50,
            "width": 50,
          }
        }
      />
      <p>
        Scopri il Digital Store
      </p>
    </styled.div>
  </TutorialButton>
</styled.div>
`;
